### DISCUSSION TITLE
Increase scroll button size in song select (its hard to click with a pen)

### DISCUSSION THREAD

**Discussion by MangoDragon** [ppy/osu#24095] [Ideas] [answered]
Currently, the more songs you have, the smaller the scroll button gets (in song select), until its just a little tiny circle. Can the shape be a constant size that's a little wider and more tall? Also, can it appear over the top of the osu! logo instead of disappearing behind it? Thank you.
![scroll](https://github.com/ppy/osu/assets/35911099/f88559dd-20b7-4e65-b12f-2676d0403fa2)

**Comment by peppy** [comment]
I agree it should not reach this small a size.

Note that you can also drag to scroll, use page up / page down, or hold right click if you turn on absolute scrolling in settings.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/24097
